content/llm/updates.md

```
@@ -4424,3 +4450,30 @@ Happy coding! 🐟
 
 ---
 
+## June 4, 2025
+
+Changes from recent development in Speaking Bots
+
+### Source: ./content/docs/updates/speaking-bots-2025-06-04.mdx
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)